app/routes/table.$.tsx

```diff
@@ -4,7 +4,7 @@ import { redirect } from '@remix-run/node'
 export const loader = (context: LoaderFunctionArgs) => {
   handleRedirects(context)
 
-  return redirect('/table/v8')
+  return redirect('/table/latest', 301)
```

Comment:
Are we sure about having a permanent redirect here? I think some browsers cache 301 redirects indefinitely, so if a user visits a `/table/*` route, that is not matched by other routes today then they might be forever (I mean indefinitely 😄 ) redirected to `/table/latest` even after we add the page they were looking for.

This would happen to `/table/latest/something`:
```
❯ curl -I  http://localhost:3000/table/latest/something
HTTP/1.1 301 Moved Permanently
location: /table/latest
Date: Wed, 24 Jan 2024 20:15:14 GMT
Connection: keep-alive
Keep-Alive: timeout=5
```

I'd go with `302`, just to be safe.

Reply:
Makes sense, I just copied what the other projects had in this file

Reply:
Hmmm, I guess this is not a new problem then... 😆 Should we do something about it, or is it a lost cause already?

Reply:
Done!